src/view/seed-success.js

```diff
@@ -40,13 +40,15 @@ const SeedSuccessView = ({ nav }) => (
           }
         </CopyText>
       </View>
-      <GlasButton onPress={() => nav.goNewAddress()}>Add some coin</GlasButton>
+      <GlasButton onPress={() => wallet.addInitialCoin()}>
```

Comment:
Update to renamed `initInitialDeposit`.

Reply:
Nice catch :).

Reply:
Jeez, must've been tired. fixed, thanks :)